pkg/cmd/cli/cmd/deploy.go

```diff
@@ -138,11 +138,7 @@ func (o *DeployOptions) Complete(f *clientcmd.Factory, args []string, out io.Wri
 	o.out = out
 
 	if len(args) > 0 {
```

Comment:
We already do; see https://github.com/openshift/origin/blob/c4c01507f6d1222e3284835081295b6ff030ebf5/pkg/cmd/cli/cmd/deploy.go#L121-L123